framework/packages/abstract-sdk/Cargo.toml

```diff
@@ -11,7 +11,7 @@ resolver = "2"
 
 # for quicker tests, cargo test --lib
 [features]
-default = []
+module-ibc = ["abstract-std/module-ibc"]
```

Comment:
Since we feature flagged it inside `abstract-std` and `abstract-sdk` we need to add feature flag inside `ibc-host` and `ibc-client`, because messages will be different depending on this feature flag